### PR TITLE
Created PT-BR translation

### PR DESCRIPTION
**Pull request by camposdelima**
I have created the messages.properties file to Brazil Portuguese. I don't translated all messages, but intend to in the future.

### CODE REVIEW

File: rundeckapp/grails-app/i18n/messages_pt_BR.properties [hunk 1]
```diff
@@ -47,22 +47,22 @@ default.paginate.ff=»
...
-gui.menu.Workflows=Jobs
-gui.menu.Events=Activity
-gui.menu.Nodes=Nodes
-gui.menu.Adhoc=Commands
-gui.menu.Admin=Configure
-gui.menu.ProjectSettings=Project Settings
-gui.menu.Projects=Projects
-gui.menu.PasswordUtility=Password Utility
-gui.menu.Users=User Summary
+gui.menu.Workflows=Procedimentos
```
Comment (carlosrfranco) [inline rundeckapp/grails-app/i18n/messages_pt_BR.properties]: Perhaps the best translation for this case could be "tarefa"

File: rundeckapp/grails-app/i18n/messages_pt_BR.properties [hunk 2]
```diff
@@ -712,11 +712,11 @@ page.notfound.message=The page you requested could not be found.
 delete.project=Delete Project
 you.are.now.logged.out=You are now logged out.
 login.again=Log In Again
-elapsed.time=Elapsed Time\:
-edit.job=Edit Job
+elapsed.time=Tempo passado\:
+edit.job=Editar job
```
Comment (carlosrfranco) [inline rundeckapp/grails-app/i18n/messages_pt_BR.properties]: follow the standard for translation of "job"

File: rundeckapp/grails-app/i18n/messages_pt_BR.properties [hunk 3]
```diff
@@ -1358,18 +1358,18 @@ menu.logStorage.button.clear.queue=Clear
 menu.logStorage.localFilesPresent.true.title=All Local log files are present on disk and can be uploaded.
 menu.logStorage.localFilesPresent.false.title=One or more local log files are not present on disk, log storage may fail.
 scheduledExecution.scheduled.cluster.orphan.title=Orphaned Schedule: This job is not owned by any cluster node, you must reschedule it by changing the owner (e.g. re-save it on this node).
-page.home.section.project.title={0} Project
-page.home.section.project.title.plural={0} Projects
-page.home.duration.in.the.last.day=In the last day
+page.home.section.project.title=Projetos {0}
```
Comment (carlosrfranco) [inline rundeckapp/grails-app/i18n/messages_pt_BR.properties]: needs to be in the singular

**Comment by camposdelima** [comment]
I dont think that we need accentuation right now. Other languages dont have so many accentuation, like PT-BR, and because that, we can create some bugs with encoding and charset in the PT-BR UI.

**Comment by carlosrfranco** [comment]
You can use ascii code for accentuation

**Comment by carlosrfranco** [comment]
Hey, Cristian,
I approved your changes and so I can finish the translation here. Thanks for the contribution.

**Pull request merged**